tests/common/dualtor/tunnel_traffic_utils.py

```diff
@@ -237,6 +237,12 @@ def _disassemble_ip_tos(tos):
             inner_dscp, _ = _disassemble_ip_tos(inner_tos)
             logging.info("Outer packet DSCP: {0:06b}, inner packet DSCP: {1:06b}".format(outer_dscp, inner_dscp))
             check_res = []
+            # For Nvidia platforms, queue check for outer/inner dscp 2/2 and 6/6 will fail due to the diversity
+            # in dscp remapping. Since we don't expect such packets in production, skip the queue check in this case.
+            if 'spc' in self.standby_tor.get_asic_name():
```

Comment:
would be more elegant to define a new function(if such does not exists) - is_nvidia_platform?

Reply:
Done.